components/application-authenticators/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -176,15 +176,14 @@ protected String getUserInfoEndpoint(OAuthClientResponse token, Map<String, Stri
         Map<ClaimMapping, String> claims = new HashMap<>();
 
         try {
-
             String accessToken = token.getParam(OIDCAuthenticatorConstants.ACCESS_TOKEN);
             String url = getUserInfoEndpoint(token, authenticatorProperties);
-
             String json = sendRequest(url, accessToken);
 
             if (StringUtils.isBlank(json)) {
                 if(log.isDebugEnabled()) {
-                    log.debug("Unable to fetch user claims. Proceeding without user claims");
+                    log.debug("Empty JSON response from user info endpoint. Unable to fetch user claims." +
```

Comment:
shouldn't we add some more information like url and access token for tracking purpose?